components/dashboard/src/CookieBanner.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { setCookie } from "./consent-cookie";
+
+function CookieBanner(props: {
+    onOpenCookieSettings: () => void;
+    onAcceptAllCookies: () => void;
+    onRejectAllCookies: () => void;
+}) {
+    const onAccept = () => {
+        setCookie(true, true);
+        props.onAcceptAllCookies();
+    };
+
+    const onReject = () => {
+        setCookie(false, false);
+        props.onRejectAllCookies();
+    };
+
+    return (
+        <div className="flex justify-between items-center mx-auto h-12 px-4 py-2 text-center text-xs text-gray-600 bg-sand-dark w-full bottom-0 left-0 fixed">
+            <p className="text-gray-600">
+                The website uses cookies to enhance the user experience. Read our{" "}
```

Comment:
Hmm, since this will explain our usage across both website and dashboard, we should reflect it in a more unified way. How about "We use cookies" or "The website and dashboard..." cc @MircoatGitpod

Reply:
> "We use cookies"

I like this one :point_up:

Reply:
or yet another alternative: "Gitpod uses cookies"